### ISSUE TITLE
Feature Idea: Configurable comment sort order

### ISSUE THREAD

**Issue by wuwinson** [mattermost/focalboard#3192]
## Summary

Ability for user to set the sort order of comments on a card. Either `oldest comments first` or `newest comments first`.

## How important this is to me and why

Importance: Medium

Use cases:
1. Allow users to see comments on a card based on their preference of seeing the newest/oldest on top.

## Additional context/similar features

- See [related thread](https://community.mattermost.com/core/pl/5j9616rwjpf97ycfews9y6z6qh) for details.
- Related ticket: https://github.com/mattermost/focalboard/issues/479

**Comment by Pinjasaur** [comment]
I was thinking about exactly this today (glad I searched for an existing issue first). Currently, comments are sorted in reverse-chronological (newest first), as per #1985. The main motivation I see in viewing comments in chronological order is keeping parity with Channels. Otherwise, newer comments are typically more relevant so having them display first can be handy UX.

Open questions:
- Would the sort order be persisted to the card or would it revert back to default upon refreshing the page or (re-)viewing a card?
- If it _is_ persisted, is it per-card, per-board, per-user, etc? And of course, _where_ would the data itself get persisted to?

cc @asaadmahmood would be interested in your thoughts

Loosely related to #343 (swapping the order of a card contents and the comments).

**Comment by wuwinson** [comment]
I would expect it to be per-user, similar to how we treat categories on the sidebar.

**Comment by asaadmahmood** [comment]
@Pinjasaur Yup, expect it to be per user, and be preserved on a card, by default it can be newest first, and then we can allow users to override that setting per card. (similar to facebook posts).

If we feel like users have a certain way of consuming threads and want all threads to be modified, then this needs to be a user setting in the settings modal.

**Comment by dreyTee** [comment]
I vote up this idea as it's unituitive to read from the bottom.
especially if comments are not oneliners.

Pretend:
you read the bottom  comment line1, line2, line3, end, 
you jump to the first line of the next comment upwards all skipping read lines, comment2 line1, line2 ...

too much unnecessary jumping.

Please make it switchable!
